site/content/docs/v1.6/contributions/minio.md

```diff
@@ -40,19 +50,11 @@ of the Velero repository is under active development and is not guaranteed to be
 
 1. Move the `velero` binary from the Velero directory to somewhere in your PATH.
 
-### MacOS Installation
-
-On Mac, you can use [HomeBrew](https://brew.sh) to install the `velero` client:
-
-```bash
-brew install velero
-```
-
 ## Set up server
 
-These instructions start the Velero server and a Minio instance that is accessible from within the cluster only. See [Expose Minio outside your cluster][31] for information about configuring your cluster for outside access to Minio. Outside access is required to access logs and run `velero describe` commands.
+These instructions start the Velero server and a Minio instance that is accessible from within the cluster only. See [Expose Minio outside your cluster](https://velero.io/docs/v1.6/contributions/minio/#expose-minio-outside-your-cluster-with-a-service) for information about configuring your cluster for outside access to Minio. Outside access is required to access logs and run `velero describe` commands.
```

Comment:
you dont need the full link here, just the `#expose-minio-outside-your-cluster-with-a-service` since the link is to a place on the same page. This makes it so the link isn't version specific. This is important especially when making changes in the main folder so that when we release new versions the links still work.